pyecharts/template.py

```diff
@@ -8,8 +8,29 @@
 import codecs
 from jinja2 import Environment, FileSystemLoader
 import pyecharts.constants as constants
+import json
 
 
+def get_resource_dir(folder):
+    """
+
+    :param folder:
+    :return:
+    """
+    current_path = os.path.dirname(__file__)
+    resource_path = os.path.join(current_path, folder)
+    return resource_path
+
+
+#with codecs.open(os.path.join(get_resource_dir('templates'), 'js', 'echarts', 'registry.json'), 'r', 'utf-8') as f:
```

Comment:
tset statement?

Reply:
Will correct it. wait for my next commit.